fitInTests/ScenarioTest.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ScenarioTest.swift
+//  fitInTests
+//
+//  Created by Kevin C on 2017-11-03.
+//  Copyright © 2017 AJ productions. All rights reserved.
+//
+
+import XCTest
+@testable import fitIn
+
+class ScenarioTest: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+    
+    func testPerformanceExample() {
+        // This is an example of a performance test case.
+        self.measure {
+            // Put the code you want to measure the time of here.
+        }
+    }
+    func testCreateScenario(){
```

Comment:
replace this with the type checking, checking for default values is going to cause headaches when I want to change the default values.